### DISCUSSION TITLE
Persisting Data

### DISCUSSION THREAD

**Discussion by HAEKADI** [community/community#25657] [Actions]
<p>Hello,</p>
<p>I have a question about storage in Github Actions.</p>
<p>Reading through the <a href="https://docs.github.com/en/actions/reference/usage-limits-billing-and-administration#artifact-and-log-retention-policy" rel="noopener nofollow ugc">doc</a>, I don’t understand if the 90 days limit retention policy is applied to everything, meaning do I lose the history of all my runs or just limited to the artifacts ?</p>

**Comment by mickeygousset** [comment, marked as answer]
<p>You can see that the workflow ran, but the detailed logs are not available. Example here: <a href="https://github.com/devopselvis/my-github-actions-demos/runs/2185174703?check_suite_focus=true" class="inline-onebox">Update demo5.yml · devopselvis/my-github-actions-demos@9f3f295 · GitHub</a></p>
<p>So yes, it also applies to your logs. You’ll still see in the Actions tag that it ran, but you can’t access the detail logs or artifacts.</p>

**Comment by Simran-B** [comment]
<p>When say history, do you refer to the <strong>log</strong> of a workflow run?<br>
Yes, it expires as well:</p>
<p><img src="https://user-images.githubusercontent.com/7819991/181092386-3728888d-31f9-48c4-b34e-f26ccd65a152.png" alt="image" data-base62-sha1="jN4ESknqnqS3qrFpq97q0ft3uT1" width="532" height="121"></p>
<p>The workflow run will still be listed on the Actions tab and the summary will remain accessible, including the list of jobs, but none of their details.</p>
<p>Packages or releases you create with a workflow do not expire, however.</p>
